Source/Logger.swift

```diff
@@ -46,6 +46,18 @@ open class Logger {
     public enum ExecutionMethod {
         case synchronous(lock: NSRecursiveLock)
         case asynchronous(queue: DispatchQueue)
+
+        func perform(work: @escaping () -> Void) {
```

Comment:
Can we make this public? Client code that uses this enum could benefit from this convenience too.

Reply:
yep!